examples/tutorials/An_Introduction_to_Polymers_and_Their__Representation.ipynb

```diff
@@ -0,0 +1,1465 @@
+{
```

Comment:
<p>Plt imported above. I'd suggest importing everything together in one cell at the top, after the pip install</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3930/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
I imported the libraries in which part it's required so that I explain why I used certain library at the first place. The imports at the top might not be easy to explain the usage part. But if it's the convention for deepchem then I will change the code.